container/trie/testdata/fuzz/FuzzSparseMerkleTrie_HashTreeRoot/5838cdfae7b16cde2707c04599b62223e7bade8dafdd8a1c53b8f881e8f79d99

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
Is the intention to commit and push these?

Reply:
Yes. These are basically regression tests. These test files produced panics that I had to fix in this PR.